### ISSUE TITLE
Tutorial on how to train a Crossencoder

### ISSUE THREAD

**Issue by alejandrojcastaneira** [facebookresearch/BLINK#126]
Hi guys! thank you for the amazing work! following the guide #116 provided by @abhinavkulkarni I was able to train a biencoder and everything looks good, however, I am not able to replicate this for a Crossencoder.

I try following the steps described [in this guide](https://github.com/facebookresearch/BLINK/tree/main/examples/zeshel):

1- train a biencoder
2- get your data into zeshel format
3- eval biencoder and generate top 64 candidates 
4- train the crossencoder based on the output of the previous step.

After step number 4, the training process starts but the loss is not decreasing nor the accuracy in the evaluation set, this remains the same after 10 epochs, so it seems the model is not being finetuned.

If someone has some experience that could share about how to finetune the crossencoder, it would be highly appreciated! thanks.

**Comment by gusevski** [comment]
Hello!
If someone have updates on this question, share please, have the same issue 
@alejandrojcastaneira, do you have any updates?)
